ci/ciimage/build.py

```diff
@@ -143,13 +143,14 @@ def copy_meson(self) -> None:
         shutil.copytree(
             self.meson_root,
             self.temp_dir / 'meson',
+            symlinks=True,
```

Comment:
@ArsenArsen your #10594 triggered this bug. :P

Reply:
I see that as my change preventing it from manifesting later :^)

Reply:
Oh, absolutely. :)